extensions/interactions/FractionInput/directives/fraction-input-rules.service.ts

```diff
@@ -34,6 +34,22 @@ export interface FractionAnswer {
 export class FractionInputRulesService {
   constructor(private fractionObjectFactory: FractionObjectFactory) {}
 
+  private isEquivalent(a, b): boolean {
```

Comment:
Added this function to compare objects. I was using `JSON.stringify(answer).toLowerCase() === JSON.stringify(
inputs.f).toLowerCase()` as a replacement of `angular.equal(answer, inputs.f)` but the problem is it works fine only if the ordering of keys in both the object is same.

cc/ @kevinlee12

Reply:
This is fine with me, but don't forget to type the args!